CppExport/CppExport.pro

```diff
@@ -15,14 +15,42 @@ HEADERS += src/precompiled.h \
 	src/dependency_analysis/DependencyUnit.h \
 	src/dependency_analysis/DependencyTarget.h \
 	src/Config.h \
-	src/dependency_analysis/DependencyComposite.h
+	src/dependency_analysis/DependencyComposite.h \
+    src/exporter/CppExporter.h \
```

Comment:
Please do 2 things:
- Fix the format
- Add all of these files to the CMakeLists.txt in the same directory (and remember to do this from now on)